plonetheme/barceloneta/overrides/Products.CMFPlone.controlpanel.browser.types.pt

```diff
@@ -27,40 +31,42 @@
             <input type="hidden" name="old_type_id" tal:attributes="value type_id" />
 
             <fieldset>
-                <select name="type_id" onchange="form.submit()">
-
-                    <tal:option repeat="selectable view/selectable_types">
-                        <option tal:content="selectable/title" i18n:translate=""
-                                tal:attributes="value selectable/id;
-                                                selected python:type_id == selectable['id'] and 'selected' or None">
-                                Content type
+                <div class="field form-group">
+                    <select name="type_id" class ="custom-select select-widget required choice-field" onchange="form.submit()">
+
+                        <tal:option repeat="selectable view/selectable_types">
+                            <option tal:content="selectable/title" i18n:translate=""
+                                    tal:attributes="value selectable/id;
+                                                    selected python:type_id == selectable['id'] and 'selected' or None">
+                                    Content type
+                            </option>
+                        </tal:option>
+                        <option
+                            value=""
+                            i18n:translate="label_default_type"
+                            tal:attributes="selected python:type_id == '' and 'selected' or None">
+                        (Default)
                         </option>
-                    </tal:option>
-                    <option
-                        value=""
-                        i18n:translate="label_default_type"
-                        tal:attributes="selected python:type_id == '' and 'selected' or None">
-                      (Default)
-                    </option>
-                </select>
-                <noscript>
-                    <input type="submit"
-                           name="form.button.SelectContentType"
-                           class="standalone"
-                           value="Change"
-                           i18n:attributes="value label_change;" />
-                </noscript>
-
-               <tal:block tal:condition="python:type_id!=''">
-
-                   <p class="discreet"
-                      tal:content="view/selected_type_description"
-                      tal:condition="view/selected_type_description"
-                      i18n:translate="">
-                       Type description
-                   </p>
-
-                   <div class="field">
+                    </select>
+                    <noscript>
+                        <input type="submit"
```

Comment:
change input submit to button submit - example in https://github.com/plone/plonetheme.barceloneta/blob/barceloneta-lts-bootstrap-base/plonetheme/barceloneta/overrides/Products.CMFPlone.controlpanel.browser.redirects-controlpanel.pt#L138